code/modules/mining/equipment/kinetic_crusher.dm

```diff
@@ -404,7 +404,7 @@
 	desc = "The blaster tubes from a colossus's arm. Suitable as a trophy for a kinetic crusher."
 	icon_state = "blaster_tubes"
 	gender = PLURAL
-	denied_type = /obj/item/crusher_trophy/blaster_tubes
```

Comment:
// Skyrat changes

Reply:
Oh wait - Fuck i meant to modify that in modular. I'll fix it promptly.

Reply:
Fixed.